### ISSUE TITLE
go to npm?

### ISSUE THREAD

**Issue by xerenade** [ajlkn/skel#171]
what about go to npm?

**Comment by XanderLuciano** [comment]
I like this idea

**Comment by ajlkn** [comment]
Good idea. Too bad `skel` is taken already, but `skel-framework` works. I'll look into it.

(PS: sorry for the delay; busy af :)

**Comment by fuzzzerd** [comment]
Is this the official package? https://www.npmjs.com/package/skel-framework-npm

**Comment by XanderLuciano** [comment]
@fuzzzerd No, it is not an official package, but rather a fork that looks to kept up to date still - albeit a few months behind Aj.

You can view the repo for that package here: https://github.com/sahellebusch/skel-framework-npm
